### PR TITLE
games-misc/parsec-bin: New Package

### PR DESCRIPTION
**Pull request by Odelpasso**
Parsec app allow to stream video games from a Windows PC to others devices
like Linux/Android or Raspberry Pi

Bug: https://bugs.gentoo.org/671332
Signed-off-by: Maxime Lombard <berillions@gmail.com>
Package-Manager: Portage-2.3.52, Repoman-2.3.12

### CODE REVIEW

**Comment by Odelpasso** [comment]
No news about this PR ?
There is someone here ? :o

**Comment by juippis** [comment]
Hey, 

there's a queue and lots of work in here ;)

Look like you included two separate PR's in this one. You should use branches in your local git tree, so you can open multiple different PR's without them affecting one another. I'll leave few comments for you, so please consider them in however you wish to fix this PR.

File: games-board/xboard/Manifest [hunk 1]
```diff
@@ -1 +1 @@
-DIST xboard-4.8.0.tar.gz 3691531 BLAKE2B 7743b654e8bdd37672a6076ce032fcc92c846796a8cf84906ae8b74454297f0fed1bce078883e9aff6237c8bfee67db659bdd7b93acd94e1b27bc90629dc3b1a SHA512 fc3231f170baacdf97df5d08488f5a1fb9a144810f7b0eb72e3e95d345ce47aa2ff67a586ef45b6a9e45932300babc59c161a29e055dd0d19b133ba986985c81
+DIST xboard-4.9.1.tar.gz 4015017 BLAKE2B c886cb06d9b4ed394220f133915a55e1dbe29d317891d00d880c597abe0ad43a8377c926d844318b0092120aee7e40fe8c6c62fff41890c35bd768575ccef731 SHA512 7c6f17450edc3f56d5987f6363da01633850b87b0d6664524a4697216dc66aa58f84314ba57ee68aebfbb01ef96995e34be4e98fb4a5124e3f5090690f91ae08
```
Comment (juippis) [inline games-board/xboard/Manifest]: Please, don't use `git mv`. Making a new ebuild in one commit, then deleting old in 2nd commit is the right way.

**Comment by Odelpasso** [comment]
There are many errors. I close this PR to do proper PR for each application.

**Pull request closed without merging**